src/components/common/Actions.vue

```diff
@@ -164,65 +164,70 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
    * @returns a promise (ie, this is an async method)
    */
   private async onClickFilePay (): Promise<void> {
-    // prevent double saving
-    if (this.isBusySaving) return
-    this.setIsFilingPaying(true)
+    // Prompt Step validations
+    this.setValidateSteps(true)
 
-    if (this.effectiveDate && !this.isValidDateTime(this.effectiveDate)) {
-      this.setIsIncorporationDateTimeValid(false)
-      this.setIsFilingPaying(false)
-      window.scrollTo({ top: 1250, behavior: 'smooth' })
-      this.setIsFilingPaying(false)
-      return
-    }
+    if (this.isEnableFilePay) {
+      // prevent double saving
+      if (this.isBusySaving) return
+      this.setIsFilingPaying(true)
 
-    // If this is a named company IA, validate NR before filing submission. This method is different
-    // from the processNameRequest method in App.vue. This method shows a generic message if
-    // the Name Request is not valid and clicking ok in the pop up redirects to the Manage Businesses
-    // dashboard.
-    if (this.isNamedBusiness) {
+      if (this.effectiveDate && !this.isValidDateTime(this.effectiveDate)) {
+        this.setIsIncorporationDateTimeValid(false)
+        this.setIsFilingPaying(false)
+        window.scrollTo({ top: 1250, behavior: 'smooth' })
```

Comment:
if you want to scroll to a specific element you can use `element.scrollIntoView({ behavior: 'smooth' })`. It is safer than put a fixed number. In mobile or in smaller views the height may be different.